packages/components/src/Status/Status.component.js

```diff
@@ -50,19 +52,46 @@ export function getbsStyleFromStatus(status) {
 			return 'muted';
 		case STATUS.WARNING:
 			return 'warning';
+		case STATUS.SKELETON:
+			return 'skeleton';
```

Comment:
this is strange to declare constant to the write strings ... but consistency FTW :D

Reply:
:D